sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -27,6 +29,8 @@ import org.apache.spark.sql.catalyst.util.ArrayData
 import org.apache.spark.sql.types._
 import org.apache.spark.unsafe.types.{ByteArray, UTF8String}
 
+import scala.util.control.NonFatal
```

Comment:
`./dev/scalastyle` do give the errors. 

I just use IDEA to do the compile job, it seems this script does not be called. I'll manually call it next time. Thanks.